### PR TITLE
BLT-8: add master templates.

### PR DESCRIPTION
**Pull request by sfrene**
Templating blocks and menus. Reorganize CSS/SCSS. Install & config module drupal/menu_item_attributes.

### CODE REVIEW

**Comment by sfrene** [comment]
Small theming issue with block--tkd-base-site-branding to do.

File: config/envs/dev/devel.settings.yml [hunk 1]
```diff
@@ -8,6 +8,6 @@ error_handlers:
 rebuild_theme: false
 debug_mail_file_format: '%to-%subject-%datetime.mail.txt'
 debug_mail_directory: 'temporary://devel-mails'
-devel_dumper: default
+devel_dumper: var_dumper
```
Comment (sharique) [inline config/envs/dev/devel.settings.yml:11]: In devel settings, do changes in your local only. don't commit it.

File: web/themes/custom/tkd_base/tkd_base.libraries.yml [hunk 2]
```diff
@@ -1,15 +1,20 @@
 global:
   css:
+    #base:
+      #dist/components/base/_footer.css: {}
+      #dist/components/andre/andre--custom.css: {}
     theme:
       dist/components/global.css: {}
+      #dist/components/andre/main.min.css: { minified: true }
+      #dist/components/andre/main--custom.css: { minified: true }
```
Comment (sharique) [inline web/themes/custom/tkd_base/tkd_base.libraries.yml:9]: remove commented code.

File: web/themes/custom/tkd_base/tkd_base.libraries.yml [hunk 3]
```diff
@@ -1,15 +1,20 @@
 global:
   css:
+    #base:
+      #dist/components/base/_footer.css: {}
+      #dist/components/andre/andre--custom.css: {}
     theme:
       dist/components/global.css: {}
```
Comment (sharique) [inline web/themes/custom/tkd_base/tkd_base.libraries.yml:7]: This file is minified.

**Comment by sharique** [comment]
@sfrene merged PR, raise separate PR for addressing review comments.

**Pull request merged**